calls.go

```diff
@@ -55,9 +55,9 @@ func calls() cli.Command {
 						Usage: "'stop' timestamp",
 					},
 					cli.Int64Flag{
-						Name:  "per-page",
+						Name:  "n",
 						Usage: "number of calls to return",
-						Value: int64(30),
+						Value: int64(100),
```

Comment:
0 and use API default like apps?

Reply:
this is no longer `--per-page`, so the API would return default number of objects per page, so, i think this should remain as is (defaulted to `100`)